Dockerfile.beaglebone-black

```diff
@@ -1,18 +1,43 @@
-FROM resin/beaglebone-debian:jessie
+#############
+# Build image
+#############
+FROM resin/beaglebone-black-alpine:3.6-20170623 AS build
 
-# Install the resinhup dependencies
-RUN apt-get update && apt-get install wget vim btrfs-tools mtools dosfstools python3 python3-urllib3 python3-sh python3-parted python3-binaryornot jq -y && rm -rf /var/lib/apt/lists/*
+# Install build requirements, create modules, then remove superfluous requirements
+## Note:
+# apk add is two lines: first is runtime dependencies, second is build time dependencies that we can remove after install
+RUN apk add --no-cache \
+      'python3<3.7' gcc libc-dev parted-dev python3-dev
```

Comment:
Removed that requirement, as I thought it was needed for the later COPY not to fail silently, but actually it does not fail silently, if python version changes, the build would bail out at the COPY step. Removed this version requirement!